libs/components/colorpicker/src/lib/modules/colorpicker/colorpicker.component.ts

```diff
@@ -546,8 +546,9 @@ export class SkyColorpickerComponent implements OnInit, OnDestroy {
         autoFitContext: SkyAffixAutoFitContext.Viewport,
         enableAutoFit: true,
         horizontalAlignment: 'left',
+        verticalAlignment: 'bottom',
         isSticky: true,
-        placement: 'below',
+        placement: 'right',
```

Comment:
Can we avoid changing the expected placement behavior and only apply the CSS fix?

Reply:
I reverted the `placement` change, but I found the `verticalAlignment` setting important to get affixer to land on an acceptable position.